dev/breeze/doc/05_test_commands.rst

```diff
@@ -187,34 +173,136 @@ Run all non-DB tests:
 
 .. code-block:: bash
 
-   breeze testing non-db-tests
+   breeze testing core-non-db-tests
 
 Only run non-DB tests from "API CLI WWW" test types:
 
 .. code-block:: bash
 
-   breeze testing non-db-tests --parallel-test-types "API CLI WWW"
+   breeze testing core-non-db-tests --parallel-test-types "API CLI WWW"
 
 Run all non-DB tests excluding those in CLI and WWW test types:
 
 .. code-block:: bash
 
-   breeze testing non-db-tests --excluded-parallel-test-types "CLI WWW"
+   breeze testing core-non-db-tests --excluded-parallel-test-types "CLI WWW"
+
+Here is the detailed set of options for the ``breeze testing core-non-db-tests`` command.
+
+.. image:: ./images/output_testing_core-non-db-tests.svg
+  :target: https://raw.githubusercontent.com/apache/airflow/main/dev/breeze/images/output_testing_core-non-db-tests.svg
```

Comment:
I don't know if it is "clever" or has been linke this in other locations... do we really want to link to github / raw on main? Why are we not using the images generated by pre-commit locally with a realitve reference?

Reply:
We are. This is how you refer to those images on GitHub I am afraid. Maybe we can do it differently, but this **works**